src/string.cr

```diff
@@ -2149,19 +2149,19 @@ class String
   # "Developers! " * 4
   # # => "Developers! Developers! Developers! Developers!"
   # ```
-  def *(times : Int)
-    raise ArgumentError.new "negative argument" if times < 0
+  def *(times count : Int)
```

Comment:
ditto. 

And in this case, this change is just a param rename. I get the consistency, but, is there any reason to prefer times over count in both cases?